public/owf.webmanifest

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Nvm, I found it in static! I assume it was just moved manually instead of `git mv` then.

Reply:
Yeah, just bad copying on my part :sweat_smile: 
Usually works to just copy manually and i will be detected correctly, but not now since i changed the file as well